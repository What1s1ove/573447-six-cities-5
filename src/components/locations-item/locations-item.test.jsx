import * as React from 'react';
import {render} from 'enzyme';
import LocationsItem from './locations-item';

describe(`LocationsItem component`, () => {
  it(`should render LocationsItem component`, () => {
    const component = render(
        <LocationsItem
          location="Amsterdam"
          isActive={false}
          onLocationChange={jest.fn()}
        />
    );

    expect(component).toMatchSnapshot();
  });
});
