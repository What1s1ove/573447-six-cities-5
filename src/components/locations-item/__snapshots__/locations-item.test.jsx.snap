// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LocationsItem component should render LocationsItem component 1`] = `
<li
  class="locations__item"
>
  <button
    class="locations__item-link tabs__item"
  >
    <span>
      Amsterdam
    </span>
  </button>
</li>
`;
